storage/objects/main_test.go

```diff
@@ -224,6 +225,60 @@ func TestKMSObjects(t *testing.T) {
 	}
 }
 
+func TestV4SignedUrl(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var (
+		bucketName = tc.ProjectID + "-signed-url-bucket-name"
+
+		objectName = "foo.txt"
+
+		serviceAccount = os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
```

Comment:
Can use the shorter variable declaration syntax and get rid of the `var()` wrapper and newlines.

```
serviceAccount := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
```